recipes/openal/all/test_package/CMakeLists.txt

```diff
@@ -4,5 +4,5 @@ project(test_package)
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup()
 
-add_executable(${PROJECT_NAME} test_package.c)
+add_executable(${PROJECT_NAME} test_package.cpp)
```

Comment:
Why did you make this change?
Did the linker complain about missing c++ symbols?
If so, then the libcxx library (`tools.stdcpp_library`). should be added to the system libraries instead of changing the linker language of test_package